tests/main.js

```diff
@@ -381,6 +381,24 @@ testMany(
   },
 )
 
+testMany('Includes includedFiles in the response of zipFunction', ['bundler_default'], async (options, t) => {
+  const { path: tmpDir } = await getTmpDir({ prefix: 'zip-it-test2' })
+  const mainFile = join(FIXTURES_DIR, 'node-module-next-image', 'function', 'function.js')
+  const result = await zipFunction(mainFile, tmpDir, {
+    ...options,
+    basePath: join(FIXTURES_DIR, 'node-module-next-image'),
+    config: {
+      '*': {
+        nodeBundler: undefined,
+        includedFiles: ['included/*.js'],
+      },
+    },
+  })
+
+  t.true(Array.isArray(result.includedFiles))
+  t.true(/node-module-next-image[/\\]included[/\\]abc\.js/.test(result.includedFiles[0]))
```

Comment:
Are you using a regular expression to catch Unix and Windows paths? If so, we've been using `unixify` to this effect:

```suggestion
  t.is(unixify(result.includedFiles[0]), 'node-module-next-image/included/abc.js')
```

Reply:
I switched this around. It is still a regex, as I cannot match the whole path as it is environment dependent. But I use unixify now.